kubernetes/cray-bos/Chart.yaml.in

```diff
@@ -0,0 +1,33 @@
+annotations:
+  artifacthub.io/images: |
+    - name: cray-boa
+      image: artifactory.algol60.net/csm-docker/stable/cray-boa:0.0.0-boa
+    - name: cray-bos
+      image: artifactory.algol60.net/csm-docker/stable/cray-bos:0.0.0-bos
+    - name: redis
+      image: docker.io/library/redis:5.0-alpine3.14
+  artifacthub.io/license: MIT
+apiVersion: v2
+appVersion: 0.0.0
+dependencies:
+- name: cray-service
+  version: ^7.0.0
```

Comment:
I applaud the use of '^'. I think it's important to be aggressive on accepting these changes.